rust/frontend/src/quota/mod.rs

```diff
@@ -340,6 +340,7 @@ pub struct QuotaExceededError {
     pub action: Action,
     pub usage: usize,
     pub limit: usize,
+    pub remediation_url: Option<String>,
```

Comment:
Suggest the #[validate()] pattern instead to ensure this is never empty by construction as opposed to inlining the check